tests/epics/test_motor.py

```diff
@@ -279,10 +279,13 @@ async def test_prepare(
     set_mock_value(sim_motor.high_limit_travel, 20)
     set_mock_value(sim_motor.max_velocity, 10)
     fake_set_signal = SignalRW(MockSignalBackend(float))
+    fake_other_set_signal = SignalRW(MockSignalBackend(float))
```

Comment:
Ermm, why are you adding this to a motor test? Could it go in `test_signal.py` instead?